reducers/storage.ts

```diff
@@ -120,7 +162,23 @@ function initialized(state = false, action: GenericAction) {
     }
 }
 
+const migrations: MigrationManifest = {
```

Comment:
Interesting, I haven't actually had a chance to use this feature of redux-persist, but it looks really convenient and easy enough to use. Hopefully we can start using redux-persist more widely to take more advantage of things like this instead of just dumping everything in `state.storage` in the future